src/utils/progressbar.js

```diff
@@ -0,0 +1,64 @@
+import React, {useState} from 'react';
+import {View, StyleSheet, Dimensions, Button, Text} from 'react-native';
+
+import ProgressBarAnimated from 'react-native-progress-bar-animated';
+
+const ProgressBar = () => {
+  const [progress, setProgress] = useState(70);
+
+  const barWidth = Dimensions.get('screen').width - 30;
```

Comment:
what's 30? magic numbers aren't cool.

Reply:
This is for syling purpose but I guess it could be moveed to "styles" if needed